test/primitives.jl

```diff
@@ -14,18 +14,18 @@ using ..TestHelpers
 
 pv1 = [(1, 2), (3, 4), (5, 6), (1, 2)]
 pv2 = [(3, 4), (5, 6), (6, 7), (3, 4)]
-lr1 = GI.LinearRing(pv1)
-lr2 =  GI.LinearRing(pv2)
-poly = GI.Polygon([lr1, lr2])
+lr1 = GO.LinearRing(pv1)
+lr2 =  GO.LinearRing(pv2)
+poly = GO.Polygon([lr1, lr2])
 
 @testset "apply" begin
 
     @testset_implementations "apply flip" begin
-        flipped_poly = GO.apply(GI.PointTrait, $poly) do p
-            (GI.y(p), GI.x(p))
+        flipped_poly = GO.apply(GO.PointTrait, $poly) do p
+            (GO.y(p), GO.x(p))
```

Comment:
Not sure this should be GO and not GI...I can see the argument for consistency though

Reply:
that should be what Geo is for though.  I don't want to advertise `GO.x` because then people will try to override it.